### PR TITLE
Add guard

### PR DESCRIPTION
**Pull request by bliof**
* https://github.com/guard/guard-cucumber/tree/v1.5.4
* https://github.com/guard/guard-rspec/tree/v4.7.3

spring rspec - does not work very well because of some errors with the
autoloading. This will be resolved in another PR.

### CODE REVIEW

**Comment by bliof** [comment]
@popovm @0mega 👀  Let's add guard 🛡

File: Guardfile [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+guard(
+  :rspec,
+  cmd: 'spring rspec',
```
Comment (bliof) [inline Guardfile]: ... this does not work all the time 😢

Comment (0mega) [inline Guardfile]: what do you mean, the tests fail?

Comment (bliof) [inline Guardfile]: There are some issues with the autoloading

```
<home>/.rvm/gems/ruby-2.3.0@shipment_tracker/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:495:in `load_missing_constant': Unable to autoload constant Events::CircleCiEvent, expected <home>/projects/shipment_tracker/app/models/events/circle_ci_event.rb to define it (LoadError)
```

**Comment by popovm** [comment]
LGTM 🐎

**Pull request merged**